### PR TITLE
Add support for multiple courses

### PR DESCRIPTION
**Pull request by alexanderMontague**
- search availability for multiple courses now! Define courses on app.js:14 like `const courses = ['CIS*3260', 'CIS*1250', ...]`
- learned you cant use es6 inside the page.evaluate callbacks but maybe we can configure that somewhere
- the data structure I stored course data in is a hash of arrays, so we can choose to display the data however we like
```
{
  'CIS*3260': [
    [
      'Fall 2020',
      'Open',
      'CIS*1250*0101 (0928) Software Design I',
      'Guelph',
      'LEC Mon, Wed, Fri\n10:30AM - 11:20AM\nAD-S, Room Virtual\nLAB Mon\n11:30AM - 01:20PM\n',
      'L. Antonie',
      '1 / 1',
      '0.50',
      'Undergraduate'
    ],
    ...
  ],
  'CIS*1250': [],
  'CIS*1234': []
}
```
The array stored per open section corresponds to a column on WA, so we can easily access stuff
image

Output with just one course:
image

Output with multiple courses:
image

TTS is still present if a course has available sections.

LMK what you think

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -5,11 +5,17 @@ const puppeteer = require('puppeteer')
 const wait = require('waait')
 
 // Time between each check in ms
-const msBetweenChecks = 120000
+const msBetweenChecks = 0
```
Comment (joshmsamuels) [inline app.js:8]: Why did you remove the sleep interval?

Comment (alexanderMontague) [reply app.js:8]: just left as part of my testing, it will default to the lowest interval possible.

Comment (joshmsamuels) [reply app.js:8]: Sounds good. Just wanted to flag it so we don’t accidentally DDOS the site
